tests/integration/scripts/test_show.py

```diff
@@ -0,0 +1,153 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import pytest
+from types import SimpleNamespace
+
+from colorama import init as colour_init
+
+from cylc.flow.id import Tokens
+from cylc.flow.scripts.show import (
+    show,
+)
+
+
+@pytest.fixture(scope='module')
+def mod_my_conf():
+    """A workflow configuration with some workflow metadata."""
+    return {
+        'meta': {
+            'title': 'Workflow Title',
+            'description': """
+                My
+                multiline
+                description.
+            """,
+            'URL': 'http://ismycomputerturnedon.com/',
+            'answer': '42',
+        },
+        'scheduling': {
+            'graph': {
+                'R1': 'foo'
+            }
+        },
+        'runtime': {
+            'foo': {
+                'meta': {
+                    'title': 'Task Title',
+                    'description': '''
+                        Task
+                        multiline
+                        description
+                    ''',
+                    'URL': (
+                        'http://hasthelargehadroncollider'
+                        'destroyedtheworldyet.com/'
```

Comment:
Had a deeply disappointing visit to that URL:
> Career and Money Advice At The Intersection Of Business And Technology

Reply:
(it used to be a static HTML page which just said "no")